packages/api/src/resolvers/recommendations/index.ts

```diff
@@ -0,0 +1,70 @@
+import {
+  CreateGroupError,
+  CreateGroupErrorCode,
+  CreateGroupSuccess,
+  MutationCreateGroupArgs,
+} from '../../generated/graphql'
+import { createGroup } from '../../services/create_group'
+import { authorized, userDataToUser } from '../../utils/helpers'
+import { homePageURL } from '../../env'
+import { getRepository } from '../../entity/utils'
+import { User } from '../../entity/user'
+
+export const createGroupResolver = authorized<
+  CreateGroupSuccess,
+  CreateGroupError,
+  MutationCreateGroupArgs
+>(async (_, { input }, { claims: { uid }, log }) => {
+  log.info('Creating group', {
+    input,
+    labels: {
+      source: 'resolver',
+      resolver: 'createGroupResolver',
+      uid,
+    },
+  })
+
+  try {
+    const userData = await getRepository(User).findOne({
+      where: { id: uid },
+      relations: ['profile'],
+    })
+    if (!userData) {
+      return {
+        errorCodes: [CreateGroupErrorCode.Unauthorized],
+      }
+    }
+
+    const [group, invite] = await createGroup({
```

Comment:
Is there a limit to the number of groups created per-user anywhere?

Reply:
Yes, just added here:
https://github.com/omnivore-app/omnivore/pull/1493/files#diff-91e988313cbb1cf4f86834197ca727146e7766377f232ee278bd4333bb0e2dd4R20-R28